### PR TITLE
[core] added missing license header to file

### LINKED ISSUE #3572: [Ttesting][Adjoint] Missing header here (with the license and the author)
https://github.com/KratosMultiphysics/Kratos/blob/c5b413682acd83040b72e26ab64eab21139bfac6/kratos/tests/strategies/schemes/test_residual_based_adjoint_bossak_scheme.cpp#L1

Besides, wait until the cpp tests are moved to a common folder

### PR DESCRIPTION
**Pull request by msandre**
addresses issue #3572 
closes #3572

### CODE REVIEW

File: kratos/tests/strategies/schemes/test_residual_based_adjoint_bossak_scheme.cpp [hunk 1]
```diff
@@ -1,3 +1,15 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:
```
Comment (loumalouomega) [inline kratos/tests/strategies/schemes/test_residual_based_adjoint_bossak_scheme.cpp:10]:
```suggestion
//  Main authors:           Michael Andre
//                                   M. Fusseder
```

**Review by loumalouomega — APPROVED** [review]
Ok, do you want to put your names there?

**Comment by msandre** [comment]
Unless it is required, I prefer to leave it blank. Git blame is more accurate imo and will tell who wrote each line.

**Comment by loumalouomega** [comment]
> Unless it is required, I prefer to leave it blank. Git blame is more accurate imo and will tell who wrote each line.

This is more related with legal reasons. It is recommended to have the reference to the license in the header of the file.

**Comment by philbucher** [comment]
@msandre FYI you can automatically close issues with PRs using some special keywords: https://help.github.com/articles/closing-issues-using-keywords/

=> this has to be done in the opening post of a PR (I edited your post)

**Comment by msandre** [comment]
@philbucher nice :+1: Another slick feature I learned in this PR is if you put a blank line in your commit msg to separate title from body, it automatically sets these as the title and body of the PR! :tada:

**Pull request merged**